### PR TITLE
Add scripts to allow working with notebooks in the columnflow environment

### PR DESCRIPTION
**Pull request by mafrahm**

### CODE REVIEW

File: notebooks/build_jupyter_kernel.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    pip install ipykernel
+
+    # install packages of interest (cf is always required)
+    pip install -r $CF_BASE/sandboxes/cf.txt
+    # pip install -r $CF_BASE/sandboxes/columnar.txt
+    pip install -r $HBW_BASE/sandboxes/ml_plotting.txt
+
+    # create the kernel
+    ipython kernel install --user --name=${DISPLAY_NAME}
```
Comment (apaasch) [inline notebooks/build_jupyter_kernel.sh:39]: Is it necessary to alwas install these or can one ask if they exist?

Comment (mafrahm) [reply notebooks/build_jupyter_kernel.sh:39]: Yes, we create a new venv from scratch, so we'll need to install all packages ourselves. You also only need to run this script once to create the kernel.
Maybe we could also just reuse the venvs created by columnflow to create a kernel, but I did not yet figure out how to do that.

Comment (apaasch) [reply notebooks/build_jupyter_kernel.sh:39]: Just to understand, your run build_jupyter_kernel.sh everytime you are in a clean shell or just once? That way you can open a jupyter Notebook, where you run the step.py script to load all modules?

Comment (mafrahm) [reply notebooks/build_jupyter_kernel.sh:39]: You run it once to create the kernel. Afterwards, you should be able to use this kernel in the jupyter notebook. The `setup` function needs to be run in each notebook to set the paths to allow importing modules from law, hbw, etc.

**Review by apaasch — APPROVED** [review]

**Pull request merged**